src/mlpack/methods/ann/layer/max_pooling.hpp

```diff
@@ -180,10 +217,10 @@ class MaxPooling
                         arma::Mat<eT>& poolingIndices)
   {
     for (size_t j = 0, colidx = 0; j < output.n_cols;
-         ++j, colidx += strideWidth)
+         ++j, colidx += strideHeight)
```

Comment:
I think @kartikdutt18, 
Has already made a PR to reflect these changes. I wonder which one of you have first seen this mistake. Let me know what both of you think about this.

Reply:
Hi @walragatver, I have already fixed this as you mentioned. I have also made the style changes that you mentioned. I fixed it related to another issue. Thanks.

Reply:
Confused me well while I was figuring our column-major ordering.